common/spec/dependabot/pull_request_creator/message_builder_spec.rb

```diff
@@ -1098,10 +1098,11 @@ def commits_details(base:, head:)
                 "<blockquote>\n"\
                 "<h2>v1.6.0</h2>\n"\
                 "<p>Mad props to <a href=\"https://github.com/greysteil\">"\
-                "@​greysteil</a> and <a href=\"https://github.com/hmarr\">"\
-                "@​hmarr</a> for the @angular/scope work - see "\
-                "<a href=\"https://github.com/gocardless/business/blob/HEAD/"\
-                "CHANGELOG.md\">changelog</a>.</p>\n"\
+                "@greysteil</a> and <a href=\"https://github.com/hmarr\">"\
```

Comment:
@feelepxyz I feel like this HTML may be incorrect.  There are two hyperlinks for the hmarr mention.

Reply:
Yes we've put a link in a link which I also think is invalid html 😬

Reply:
Oh think I know what's gone wrong, I think we need to return early in the `walk` if the node is a `:link` so we don't end up walking inside the link and finding a text node there with a mention 😅